### ISSUE TITLE
What to do and how to detect if the failed_requests file is invalid?

### ISSUE THREAD

**Issue by ErikBjare** [ActivityWatch/activitywatch#73]
Currently, if an event in the file is invalid it could try and send the event to the server indefinitely (happened to me while debugging something). Perpetually failing and never getting anywhere.

Having a way to handle this case could potentially make such events self-resolve (by skipping them if not accepted by the server due to a HTTP 400 by example).

Note the case in which the JSON can't be parsed is already handled: https://github.com/ActivityWatch/aw-client/blob/1ef887ad27b4acbe30c5b7edefe3241bd8eb6819/aw_client/client.py#L191

**Comment by ErikBjare** [comment]
I came across this earlier today when I had ran some tests.

The tests were abruptly cancelled (SIGSEGV I think) and heartbeats ended up in the queue file. When I re-ran the tests, it kept trying to send the old events to a bucket that no longer existed (since it had been removed by the test cleanup).

This problem has at least ~~two~~ three solutions:

 - Add a way clear the queue manually (which would be done at initialization)
 - Fix this issue
 - **What Johan has done here** (unmerged due to an issue): https://github.com/ActivityWatch/aw-client/pull/21
